services/collaboration/pkg/service/grpc/v0/service.go

```diff
@@ -107,6 +117,10 @@ func (s *Service) OpenInApp(
 		// there is no known case of supporting edit only without view
 		editAppURL = viewAppURL
 	}
+	if viewAppURL == "" {
```

Comment:
WOPI defaults to appurl if the appviewurl is empty.
https://github.com/cs3org/wopiserver/blob/562a6e16e00745b526c19583ed8acf5edcc3d977/src/wopiserver.py#L289-L290
```
		//    - string appurl: the URL of the end-user application
		//    - string appviewurl (optional): the URL of the end-user application in view mode when different (defaults to appurl)
		// 
```
https://github.com/cs3org/wopiserver/blob/562a6e16e00745b526c19583ed8acf5edcc3d977/src/wopiserver.py#L333
```
		// appviewurl = url_unquote_plus(req.args.get('appviewurl', appurl)).strip('/')
```